metaflow/system/system_monitor.py

```diff
@@ -49,11 +69,9 @@ def _debug(msg: str):
         msg : str
             Message to log.
 
-        Returns
-        -------
-        None
         """
-        print("system monitor: %s" % msg, file=sys.stderr)
+        if os.environ.get("METAFLOW_DEBUG_SIDECAR", "0") == "1":
```

Comment:
minor nit, why not use `debug.sidecar_exec`? The reason I ask is we handle True/0/1 in a slightly different way so it would be more consistent.

Reply:
`debug.sidecar_exec` won't work when used outside of a flow. Open to replacing with the correct check to make it more consistent.